SPM: add binding usage (#21948)
pingcap/docs: sql-plan-management.md

1 row in set (0.00 sec)
```

> ### Binding usage

Reviewer: ```suggestion
### Binding usage statistics
```

> Starting from v8.5.4 and v9.0.0, TiDB introduces the [`tidb_enable_binding_usage`](/system-variables.md#tidb_enable_binding_usage-new-in-v854-and-v900) system variable to control whether to collect the usage statistics of SQL plan bindings. 

Reviewer: ```suggestion
Starting from v8.5.5 and v9.0.0, TiDB introduces the [`tidb_enable_binding_usage`](/system-variables.md#tidb_enable_binding_usage-new-in-v854-and-v900) system variable to control whether to collect the usage statistics of SQL plan bindings. 
```

> You can use the [`tidb_enable_binding_usage`](/system-variables.md#tidb_enable_binding_usage-new-in-v900) system variable to control whether to collect the usage statistics of SQL plan bindings. 

Reviewer: ```suggestion
You can use the [`tidb_enable_binding_usage`](/system-variables.md#tidb_enable_binding_usage-new-in-v900) system variable (`ON` by default) to control whether to collect the usage statistics of SQL plan bindings. 
```

> The binding usage feature collects usage statistics of SQL plan bindings. You can use these statistics to identify unused bindings and optimize your binding management strategy, such as removing bindings that are no longer needed or adjusting existing bindings to improve query performance.

Reviewer: ```suggestion
Starting from v8.5.4 and v9.0.0, TiDB supports collecting usage statistics of SQL plan bindings. You can use these statistics to identify unused bindings and optimize your binding management strategy, such as removing bindings that are no longer needed or adjusting existing bindings to improve query performance.
```
